qiskit/circuit/commutation_checker.py

```diff
@@ -72,7 +56,7 @@ def commute(
         op2: Operation,
         qargs2: List,
         cargs2: List,
-        max_num_qubits: int = 3,
+        max_num_qubits: int = 4,
```

Comment:
I see that you have the default value from `3` to `4`. Out of curiosity, does this show much difference on "practical" circuits (like the ones we have in ASV benchmarking)? Also it may be good to add a comment about this in the release notes.

Reply:
I changed it back to 3, that preliminary change made it into my commit, sorry.